awx/main/dispatch/worker/callback.py

```diff
@@ -151,12 +151,14 @@ def flush(self, force=False):
             metrics_singular_events_saved = 0
             metrics_events_batch_save_errors = 0
             metrics_events_broadcast = 0
+            metrics_job_lag = 0
```

Comment:
probably want a `metrics_total_job_event_processing_seconds` that we add to as we loop over events